### PR TITLE
Fixed the last word pop up window instantly killing you if you are fu…

### PR DESCRIPTION
**Pull request by StudentBlossom**
…ll health

## What Does This PR Do
Fixed the last words popup being able to take the player out even if in perfect health

## Why It's Good For The Game
Prevent people from dying on accident

## Testing
I hit myself into crit, clicked on the succumb button rejuvenated me using admin power then when I clicked "ok" on the last word window I did not die

<hr>

### Declaration
- [X] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/feature_freeze/CODE_OF_CONDUCT.md#currently-changes-to-the-following-types-of-content-reuires-pre-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.

<hr>

## Changelog
:cl:
fix: Last Word Window killing perfect health players
/:cl:

### CODE REVIEW

**Review by DGamerL — APPROVED** [review]
💀

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -306,7 +306,9 @@
 	if(stat == DEAD)
 		// cancel em out if they died while they had the message box up
 		last_words = null
-
+	if(health >= HEALTH_THRESHOLD_CRIT)
+		to_chat(src, "<span class='warning'>You are unable to succumb to death! This life continues!</span>")
```
Comment (lewcc) [inline code/modules/mob/living/living.dm:310]:
```suggestion
		to_chat(src, "<span class='warning'>You are unable to succumb to death! This life continues!</span>")
		last_words = null
```

**Review by lewcc — COMMENTED** [review]
Make sure these are cleared out for the next time they die

File: code/modules/mob/living/living.dm (hunk 1 above)
Comment (DGamerL) [reply code/modules/mob/living/living.dm:310]: Isn't this always reset by 
```js
var/last_words = tgui_input_text(src, "Do you have any last words?", "Goodnight, Sweet Prince", encode = FALSE)
```

**Review by DGamerL — APPROVED** [review]
I'll fix the remaining issue in a different PR, good fix!

**Pull request merged**